lib/Field/styles.scss

```diff
@@ -47,12 +47,32 @@ $field-data-color: $neutral-base !default;
 .field__instructions {
   font-style: italic;
   margin: $field-spacing-base 0 0;
+  padding: $field-spacing-base 0 0 0;
+  border-top: 1px solid $neutral-light;
+  transition: all 0.2s ease; // Does this go here?
```

Comment:
This can go here but I wonder if we should use a setting? In `styles/settings/animations.scss` we have a setting for mirco animations at `.3s`. What do you think?

Reply:
Or we could change that setting to `0.2s`.

Reply:
Would prefer to set it to 0.2s, anytime I've used 0.3 it's felt too quick.

Reply:
Sweet, change the setting and then use it in the 2 places you've added it 👍